<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1378</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1378</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1378</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>21</y>
      <width>1021</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="elementsBox">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>50</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>20</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>элементы:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createArrButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>50</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>создать</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SortBox">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>110</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>QuickSort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MergeSort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HeapSort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>InterpolationSort</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="searchElBox">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>310</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>280</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск элемента:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchElButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>310</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>поиск</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="timeText">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>200</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>170</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Время сортировки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>350</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>BinPow</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="BinPowBox">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>380</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>110</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>начать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1378</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
